samples/retained-instance-store/src/main/java/com/badoo/ribs/samples/retained_instance_store/rib/retainedInstance/RetainedInstance.kt

```diff
@@ -0,0 +1,8 @@
+package com.badoo.ribs.samples.retained_instance_store.rib.retainedInstance
+
+import com.badoo.ribs.core.Rib
+
+interface RetainedInstance : Rib {
```

Comment:
```suggestion
interface RetainedInstanceExample : Rib {
```

(Rename with plugin)